contributing.md

```diff
@@ -0,0 +1,140 @@
+Please read through it carefully, it helps both the contributors (you guys) and maintainers be more efficient.
+
+# Step 1. Get an Invitation to the Organization
+To get assigned to an issue, you'll first have to invited by us to the organization!
+You'll then receive an email.
+Once the invite if accepted, only then we'd be able to assign you to the issue.
+To obtain an invite check out [https://github.com/wtfpython-web/wtfpython-web/issues/11](https://github.com/wtfpython-web/wtfpython-web/issues/11)
```

Comment:
Automated invite link has been activated and the URL is https://wtfpython-web-invite.herokuapp.com/ so you'll have to rephrase this sentence with the new link